process/accounts/accounts_processor_test.go

```diff
@@ -58,110 +45,58 @@ func TestNewAccountsProcessor(t *testing.T) {
 	}
 }
 
-func TestAccountsProcessor_ProcessAccounts_InvalidUserAccountHandler_ExpectZeroAccounts(t *testing.T) {
-	ap, _ := accounts.NewAccountsProcessor(
-		&mock.ShardCoordinatorMock{},
-		&mock.AccountsAdapterStub{
-			LoadAccountCalled: func(address []byte) (vmcommon.AccountHandler, error) {
-				return nil, nil
-			}},
-		&mock.PubKeyConverterStub{})
-
-	tx := &schema.Transaction{
-		Receiver: testscommon.GenerateRandomBytes(),
-		Sender:   testscommon.GenerateRandomBytes()}
-	ret := ap.ProcessAccounts([]*schema.Transaction{tx}, []*schema.SCResult{}, []*schema.Receipt{})
-
-	require.Len(t, ret, 0)
-}
-
-func TestAccountsProcessor_ProcessAccounts_InvalidLoadAccount_ExpectZeroAccounts(t *testing.T) {
-	ap, _ := accounts.NewAccountsProcessor(
-		&mock.ShardCoordinatorMock{},
-		&mock.AccountsAdapterStub{
-			LoadAccountCalled: func(address []byte) (vmcommon.AccountHandler, error) {
-				return nil, errors.New("load account error")
-			}},
-		&mock.PubKeyConverterStub{})
-
-	tx := &schema.Transaction{
-		Receiver: testscommon.GenerateRandomBytes(),
-		Sender:   testscommon.GenerateRandomBytes()}
-	ret := ap.ProcessAccounts([]*schema.Transaction{tx}, []*schema.SCResult{}, []*schema.Receipt{})
-
-	require.Len(t, ret, 0)
-}
-
 func TestAccountsProcessor_ProcessAccounts_NotInSameShard_ExpectZeroAccounts(t *testing.T) {
```

Comment:
maybe remove extra "_" ?

Reply:
no, the test names are much easier to read this way. this naming pattern is present in the entire repository

Reply:
:pepe-riot: